app/scripts/views/base.js

```diff
@@ -681,7 +670,7 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
         event.preventDefault();
       }
 
-      var args = [].slice.call(arguments, 0);
+      let args = [].slice.call(arguments, 0);
```

Comment:
Not sure if we can't just change L667 above to something like this (and avoid line 673-674):
```js
BaseView.preventDefaultThen = function (handler, ...args) {
```